source/faq.rst

```diff
@@ -52,10 +53,60 @@ How do I navigate to another Pod?
 To browse another Pod (or even a different Container in your own Pod), you can navigate there 
 directly by entering the Pod (or Container) URI.
 
-#. In PodBrowser, click the current :guilabel:`Pod` identifier link (just below the application logo).
+#. In PodBrowser, click the current :guilabel:`Pod` identifier link (just below the Account menu).
+
+#. Select :guilabel:`Change Pod` in the menu.
+
+#. In the displayed panel, enter the URI for the Pod (or Container).
+
+#. Press :guilabel:`Enter`, or click the :guilabel:`Go` button.
+
+#. The navigation panel is refreshed and shows the Resources in the entered Pod (or Container).
+
+What do the Resource permissions mean?
+======================================
+
+PodBrowser allows you to share Containers and Resources within your
+Pod with other Solid users. But what do the available permissions allow the other Solid users to do? 
```

Comment:
Since the second sentence is repeating the header question - maybe just

You can share your Pod's Containers and Resources with other Solid users.  To share, you grant to the other users specific permission(s) for the Containers and Resources. The following table describes the permissions; these permissions can be combined:

Reply:
OK